src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Commands/Checkout/ChoosePaymentMethod.xml

```diff
@@ -17,7 +17,7 @@
 >
     <class name="Sylius\Bundle\ApiBundle\Command\Checkout\ChoosePaymentMethod">
         <attribute name="paymentMethodCode" serialized-name="paymentMethod">
-            <group>shop:cart:select_payment_method</group>
+            <group>shop:order:select_payment_method</group>
```

Comment:
following the example of order item, it seems to me, that 
```suggestion
            <group>shop:payment:update</group>
```
would be a better group for this endpoint